README.md

```diff
@@ -4,9 +4,301 @@ The compliance-operator is a OpenShift Operator that allows an administrator
 to run compliance scans and provide remediations for the issues found. The
 operator leverages OpenSCAP under the hood to perform the scans.
 
-The operator runs in the `openshift-compliance` namespace, so make sure
-all namespaced resources like the deployment or the custom resources the
-operator consumes are created there.
+By default, the operator runs in the `openshift-compliance` namespace, so
+make sure all namespaced resources like the deployment or the custom resources
+the operator consumes are created there. However, it is possible for the
+operator to be deployed in other namespaces as well.
+
+## The Objects
+
+### The `ComplianceSuite` object
+
+The API resource that you'll be interacting with is called `ComplianceSuite`.
+It is a collection of `ComplianceScan` objects, each of which describes
+a scan. In addition, the `ComplianceSuite` also contains an overview of the
+remediations found and the statuses of the scans. Typically, you'll want
+to map a scan to a `MachinePool`, mostly because the remediations for any
+issues that would be found contain `MachineConfig` objects that must be
+applied to a pool.
+
+A `ComplianceSuite` will look similar to this:
+
+```yaml
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ComplianceSuite
+metadata:
+  name: fedramp-moderate
+spec:
+  autoApplyRemediations: false
+  schedule: "0 1 * * *"
+  scans:
+    - name: workers-scan
+      scanType: Node
+      profile: xccdf_org.ssgproject.content_profile_moderate
+      content: ssg-ocp4-ds.xml
+      contentImage: quay.io/complianceascode/ocp4:latest
+      rule: "xccdf_org.ssgproject.content_rule_no_netrc_files"
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+status:
+  aggregatedPhase: DONE
+  aggregatedResult: NON-COMPLIANT
+  scanStatuses:
+  - name: workers-scan
+    phase: DONE
+    result: NON-COMPLIANT
+```
+In the `spec`:
+* **autoApplyRemediations**: Specifies if any remediations found from the
+  scan(s) should be applied automatically.
+* **schedule**: Defines how often should the scan(s) be run in cron format.
+* **scans** contains a list of scan specifications to run in the cluster.
+
+In the `status`:
+* **aggregatedPhase**: indicates the overall phase where the scans are at. To
+  get the results you normally have to wait for the overall phase to be `DONE`.
+* **aggregatedResult**: Is the overall verdict of the cluster.
```

Comment:
The scan suite may not necessarily cover the cluster, so we probably want to say overall verdict of the suite.

Reply:
Good suggestion! fixed.